sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1116,6 +1117,262 @@ def max_deflection(self):
         else:
             return None
 
+    def plot_shear_force(self, var={}):
+        """
+        Returns a plot for Shear force present in the Beam object.
+
+        Parameters
+        ==========
+        var : dictionary
```

Comment:
Maybe call this `subs` instead of `var` as it resembles the `subs()` method on SymPy expressions.